lnrpc/rpc.proto

```diff
@@ -2990,6 +2990,13 @@ message Payment {
 
     /// The HTLCs made in attempt to settle the payment [EXPERIMENTAL].
     repeated HTLCAttempt htlcs = 14;
+
+    /**
+    The creation index of this payment. Each payment can be uniquely identified
+    by this index, which is monotonically increasing by creation time but can
+    have gaps due to deleted payments.
```

Comment:
Technically not due to deleted payments right now; we used to only save completed payments to the db, not failed attempts (😱 ).  This could be a bit misleading to a user, because they haven't deleted any payments, but may see gaps. I would go with, not strictly incremented by 1 for payments made in older versions of lnd, or something like that. If/when we add payment deletion, we can update the comment.

Reply:
Right, updated the comment.